File: projE5.R
#project euler 5 

# Smallest multiple


n = 20

x = seq(20,2)

res <- 0

fact = 0


while(fact != 2){
  
  res <- res+20
  
  for (i in 1 : length(x)){
    
    
    
    rem <- res%%x[i]
    if(rem != 0){ break }
    
    fact = x[i]

    
  }
    
}

File: projE5.py
# Smallest multiple


res = 0
fact = 20

while fact !=2:
    res += 20
    for i in range(19,1, -1):
        rem = res % i
        if rem != 0:
            break
        fact = i

print(f"the smallest positive number evenly divisible by all number from 1 to 20 is {res}")
